bambi/priors/link.py

```diff
@@ -90,6 +94,34 @@ def logit(mu):
 
 
 class Link:
+    """Representation of link function.
+
+    This object actually contains two main functions. One is the link function itself, the function
+    that maps values in the response scale to the linear predictor, and the other is the inverse
+    of the link function, that maps values of the linear predictor to the response scale.
+
+    The great majority of users will never interact with this class unless they want to create
+    a custom ``Family`` with a custom ``Link``. This is automatically handled for all the built-in
+    families.
+
+    Parameters
+    ----------
+    name: str
+        The name of the link function. If it is a known name, it's not necessary to pass any
+        other arguments because functions are already defined internally. If not known, all of
+        ``link``, ``linkinv`` and ``linkinv_backend`` must be specified.
+    link: function
+        A function that maps the response to the linear predictor. Known as 'g()' in GLM jargon.
+        Does not need to be specified when ``name`` is a known name.
+    linkinv: function
+        A function that maps the linear predictor to the response. Knonwn as 'g()^-1' in GLM jargon.
+        Does not need to be specified when ``name`` is a known name.
+    linkinv_backend: function
+        Same than ``linkinv`` but must be something that works with PyMC3 backend (i.e. it must
+        work with Theano tensors). Does not need to be specified when ``name`` is a known name.
```

Comment:
```suggestion
        work with Aesara tensors). Does not need to be specified when ``name`` is a known name.
```

Probably better to use the new library name

Reply:
You're right!